Gopkg.lock

```diff
@@ -953,6 +964,7 @@
     "github.com/awslabs/goformation/cloudformation",
     "github.com/emicklei/go-restful",
     "github.com/golang/mock/gomock",
+    "github.com/kylelemons/godebug/pretty",
```

Comment:
Are changes to this file expected to be part of this PR?

Reply:
erggh... definitely not. Let me revert that.

Reply:
It has been reverted.